docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -91,27 +95,60 @@ Install the iOS SDK via Swift Package Manager (SPM). There are two installation
 
 Add the following dependency to Package.swift:
 
-``` javascript
+```js
 dependencies: [
-.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "0.3.0"),
+.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "1.2.0"),
 ]
 ```
 
 ### Installing with Xcode
 
-Add the following dependency via Xcode:
+Add the following entry in the Package Dependencies for your apps:
 
-![Xcode Dependency](images/xcode-uid2-ios-sdk.png)
+| Name | Location | Dependency Rule                         |
+| :--- | :--- |:----------------------------------------| 
+| uid2-ios-sdk | git@github.com:IABTechLab/uid2-ios-sdk.git | Up to next major version: 1.2.0 < 2.0.0 |
```

Comment:
Docusaurus turns git@github.com into an email link. Should we format it as code to stop that?

Reply:
Yes. Thx. Done.